### ISSUE TITLE
[GOREPORT] Fix linting issues maya/types/v1/util.go

### ISSUE THREAD

**Issue by nsathyaseelan** [openebs/openebs#764]
Goreport is showing several linting errors. Please see the report at
https://goreportcard.com/report/github.com/openebs/maya

This issue is to fix the errors in
https://github.com/openebs/maya/blob/master/types/v1/util.go
```
Line 252: warning: comment on exported function GetOrchestratorName should be of the form "GetOrchestratorName ..." (golint)
Line 338: warning: exported function DefaultOrchestratorAddress should have comment or be unexported (golint)
Line 870: warning: comment on exported function MakeOrDefJivaControllerArgs should be of the form "MakeOrDefJivaControllerArgs ..." (golint)
Line 920: warning: comment on exported function SanitiseVSMName should be of the form "SanitiseVSMName ..." (golint)
```

**Comment by ferhatelmas** [comment]
@prateekpandey14 this should be closed.

**Comment by yudaykiran** [comment]
Closing this issue, fixed by PR #188

**Comment by ferhatelmas** [comment]
@yudaykiran please write full url because PR is on maya, not on openebs repo. Otherwise it shows wrong PR.
